linkis-dist/package/conf/linkis-ps-publicservice.properties

```diff
@@ -48,3 +48,4 @@ spring.spring.main.allow-bean-definition-overriding=true
 spring.spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
 # ps-cs prefix must be started with 'cs_'
 spring.eureka.instance.metadata-map.route=cs_1_dev
+linkis.metadata.hive.permission.with-login-user-enabled=false
```

Comment:
The above configuration `#linkis.metadata.hive.permission.with-login-user-enabled`can be removed
It's better to add a note that the default value is true